webxr/glwindow/mod.rs

```diff
@@ -390,8 +387,8 @@ impl GlWindowDevice {
                 .context_surface_info(&context)
                 .unwrap()
                 .map(|info| info.framebuffer_object)
-                .unwrap_or(0);
-            gl.bind_framebuffer(gl::FRAMEBUFFER, framebuffer(framebuffer_object));
+                .flatten();
```

Comment:
Could this and the other call sites use `and_then` to avoid the `map()` immediately followed by `flatten()` here and other places in this PR?